code/src/components/MovieDetail.js

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect, useState } from 'react';
+
+import { Link, useParams } from 'react-router-dom';
+import { API_KEY } from '../key';
+import Icon from './back_button.svg'
+
+
+export const MovieDetail = () => {
+    const { movieId } = useParams();
+    const [movies, setMovies] = useState([]);
+    const DETAILS_API = `https://api.themoviedb.org/3/movie/${movieId}?api_key=${API_KEY}&language=en-US`;
+
+    useEffect(() => {
+      fetch(DETAILS_API)
+      .then((response) => response.json())
+      .then((json) => {
+        setMovies(json)
+        })
+        .catch((error) => {
+            console.error('Request failed', error)
+        })
+  }, [movieId, DETAILS_API]);
+
+return(
+<>
+  <article className ="details-page">
+    <section className="detail-backdrop" style={{  backgroundImage: `linear-gradient(rgba(0, 0, 0, 0) 70%, rgb(0, 0, 0) 100%),  url(https://image.tmdb.org/t/p/w1280${movies.backdrop_path})` }}>
+      <div className="movie-detail-content">
+        <div className="movie-detail-back">
+          <Link to ="/" className="link" exact='true'>
```

Comment:
There is limited set of props available for `<Link />` predefined component provided by `react-router-dom` package and `exact` is not one of them - basically, it's not doing anything as it's not available for it. It's only available for `<Route />` and `<NavLink />` components.

For further reference check out [https://reactrouter.com/web/api/Link](https://reactrouter.com/web/api/Link)